docs/release-notes/8.3.asciidoc

```diff
@@ -3,8 +3,81 @@
 
 [discrete]
 [[release-notes-8.3.1]]
+
 === 8.3.1
 
+[discrete]
+[[known-issue-8.3.1]]
+==== Known issue
+
+8.3.1 has a bug where detection rules that were created or edited in 8.2 will stop running after you upgrade. Because of this, we advise against upgrading from 8.2 to 8.3.1. If you want to upgrade, _we highly recommend that you track your active rules beforehand so you can easily find and re-enable them after you upgrade_. Use the <<rules-api-find, Find rules API>> to capture a list of enabled rules before you upgrade.
```

Comment:
Is this all of `8.2.x` versions or just `8.2.0`? We should prob specify either way.

Reply:
@peluja1012 do you happen to know if this affects users on all versions of `8.2`?

Reply:
Confirmed via slack with @peluja1012  that this bug does affect users who are upgrading from any version of 8.2.